### PR TITLE
 Added confused debuff for Brain of Confusion

### PR DESCRIPTION
**Pull request by tysonstrange**

### CODE REVIEW

**Comment by hakusaro** [comment]
Hilariously, I have no idea how to add commits to your branch

```
Counting objects: 3, done.
Delta compression using up to 4 threads.
Compressing objects: 100% (3/3), done.
Writing objects: 100% (3/3), 345 bytes | 345.00 KiB/s, done.
Total 3 (delta 2), reused 0 (delta 0)
remote: Resolving deltas: 100% (2/2), completed with 2 local objects.
To github.com:tysonstrange/TShock.git
 ! [remote rejected]   tyson/general-devel -> tyson/general-devel (permission denied)
error: failed to push some refs to 'git@github.com:tysonstrange/TShock.git'
```

So I'm just going to say we can merge this and I'll fix the changelog later.

**Review by hakusaro — APPROVED** [review]
lgtm_6

Changelog needs to be fixed but that's something for me to do after merge.

**Comment by hakusaro** [comment]
@Pryaxis/tshock if y'all could review/merge that'd be great

File: TShockAPI/Bouncer.cs [hunk 1]
```diff
@@ -1869,7 +1869,9 @@ internal void OnKillMe(object sender, GetDataHandlers.KillMeEventArgs args)
 			{ BuffID.DryadsWardDebuff, 120 },
 			{ BuffID.Daybreak, 300 },
 			{ BuffID.BetsysCurse, 600 },
-			{ BuffID.Oiled, 540 }
+			{ BuffID.Oiled, 540 },
+			{ BuffID.Confused, 360 }, // Brain of Confusion Internal Item ID: 3223
+			{ BuffID.Daybreak, 300 } // Solar Eruption Item ID: 3473, Daybreak Item ID: 3543
```
Comment (AxeelAnder) [inline TShockAPI/Bouncer.cs:1874]: >{ BuffID.DryadsWardDebuff, 120 },
			**{ BuffID.Daybreak, 300 },**
			{ BuffID.BetsysCurse, 600 },
			{ BuffID.Oiled, 540 }
			{ BuffID.Oiled, 540 },
			{ BuffID.Confused, 360 }, // Brain of Confusion Internal Item ID: 3223
			**{ BuffID.Daybreak, 300 } // Solar Eruption Item ID: 3473, Daybreak Item ID: 3543**

Daybreak has been added, this causes a initialization error

**Comment by QuiCM** [comment]
Thanks @AxeelAnder - I've added a commit to gen dev that should fix the double daybreak entry

**Pull request merged**